src/dbnode/namespace/kvadmin/ns_admin.go

```diff
@@ -82,6 +82,36 @@ func (as *adminService) Delete(name string) error {
 	return ErrNotImplemented
 }
 
+func (as *adminService) ResetSchema(name string) error {
+	currentRegistry, currentVersion, err := as.currentRegistry()
+	if err == kv.ErrNotFound {
+		return ErrNamespaceNotFound
+	}
+	if err != nil {
+		return xerrors.Wrapf(err, "failed to load current namespace metadatas for %s", as.key)
+	}
+
+	var targetMeta *nsproto.NamespaceOptions
+	for nsID, nsOpts := range currentRegistry.GetNamespaces() {
+		if nsID == name {
+			targetMeta = nsOpts
+			break
+		}
+	}
+	if targetMeta == nil {
+		return ErrNamespaceNotFound
+	}
+
+	// Clear schema options in place.
+	targetMeta.SchemaOptions = nil
```

Comment:
So this will be a backward incompatible change which means if you do it to a cluster that is running it could cause really unpredictable behavior...I wonder if its best that if you really mess up the schema you just delete the namespace and move on? I guess if its just for internal usage / testing its fine but we should never tell anyone to use this

Reply:
Behavior is well defined:
Reset will not impact running dbnode, in memory state won't be updated if new schema from etcd does not extend from current one. (e.g. setting schema to nil).

Still, agree, this is only for administrator to fix schema if there is operator error. Added a big warning message and require a force header to be set to proceed.